src/utils/exr-parser.worker.js

```diff
@@ -17,102 +17,103 @@
  *  };
  */
 
-let openEXRLoaded = false;
-let queuedJobs = [];
-let OpenEXR;
+exports.__esModule = true;
 
-self.addEventListener('message', (event) => {
+var EXR = require("../exr-wrap/exr-wrap.js");
```

Comment:
This is an important change as previously the EXR was being bound as a global var I believe, much better to require it as a module like this.